pkg/argsbuilder/submit_pytorchjob.go

```diff
@@ -173,12 +174,17 @@ func (s *SubmitPytorchJobArgsBuilder) check() error {
 	return nil
 }
 
-func (s *SubmitPytorchJobArgsBuilder) addMasterAddr() error {
+func (s *SubmitPytorchJobArgsBuilder) addEnv() error {
 	if s.args.EnableRDMA {
 		if s.args.Envs == nil {
 			s.args.Envs = map[string]string{}
 		}
 		s.args.Envs["MASTER_ADDR"] = fmt.Sprintf("%v-master-0", s.args.Name)
 	}
+
+	if s.args.NprocPerNode != "" {
```

Comment:
Do we need to determine if s.args.Envs is empty pointer here?

Reply:
Done.